spring-cloud-starter-stream-sink-mqtt/src/main/java/org/springframework/cloud/stream/app/mqtt/sink/MqttSinkProperties.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.app.mqtt.sink;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Properties for the Mqtt Sink.
+ *
+ * @author Janne Valkealahti
+ *
+ */
+@ConfigurationProperties("mqtt")
+public class MqttSinkProperties {
+
+	/**
+	 * identifies the client
+	 */
+	private String clientId = "scdf.mqtt.client.id.sink";
```

Comment:
When porting from xd, it uses xd in id so followed a pattern. Wonder what it should be?